rcl/test/rcl/test_service.cpp

```diff
@@ -76,6 +77,30 @@ class CLASSNAME (TestServiceFixture, RMW_IMPLEMENTATION) : public ::testing::Tes
   }
 };
 
+void
+wait_for_server_to_be_available(
+  rcl_node_t * node,
+  rcl_client_t * client,
+  size_t max_tries,
```

Comment:
@Barry-Xu-2018 I believe `max_retries` would be more accurate with the corresponding change in the loop to fulfill that contract.

Reply:
Same below.

Reply:
@hidmic 

I think max_retries maybe make caller confusion. 
The value of max_retries is valid after trying once. So retry once is not the same as try once.  

```
ex:
    wait_for_subscription_to_be_ready(&subscription, context_ptr, 10, 100, success)

About 10,                                                                 
max_tries: it means to do something for 10 times
max_retries: it means to do something for 1 times (try) and 10 times (retry)
```

So it is better to use max_tries.  
What do you think ?

Reply:
In that case, you must ban `max_tries = 0` since this code will always at least try once.

Reply:
> In that case, you must ban `max_tries = 0` since this code will always at least try once.

Yes.  I update codes.